### PR TITLE
Readme-improvements

### LINKED ISSUE #393: [DOCS] Readme can be improved
### Description

Readme is not properly organized which makes it hard to understand, and it feels a bit cluttered.

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Dhananjaymahajan2001**
- YouTube link simplified 
- note for participants in a separate section

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/397"></a>

resolve #393

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
 # LinkFree - EddieHub
 
-An open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - built by the [EddieHub](https://www.eddiehub.org)\
-Initially created on a YouTube live stream <https://www.youtube.com/watch?v=Jorl_vcp-Ew>
+An open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - built by the [EddieHub](https://www.eddiehub.org).\
+Initially created on a [YouTube live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree)
+
+![Screenshot](https://user-images.githubusercontent.com/60853067/133296120-dbdb1799-4cca-4708-81ce-05edc65e59c9.png)
```
Comment (rohan-kulkarni-25) [inline README.md:8]: - Please add the alternative text a bit relative

File: README.md [hunk 2]
```diff
@@ -1,7 +1,16 @@
 # LinkFree - EddieHub
 
-An open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - built by the [EddieHub](https://www.eddiehub.org)\
-Initially created on a YouTube live stream <https://www.youtube.com/watch?v=Jorl_vcp-Ew>
++ An open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - built by the [EddieHub](https://www.eddiehub.org).
+
++ Initially created on a [YouTube live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).
```
Comment (Panquesito7) [inline README.md:5]: I recommend reverting these changes and putting them back as to how it was.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+
+
+ [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree)
+
+
+ ![Screenshot](https://user-images.githubusercontent.com/60853067/133296120-dbdb1799-4cca-4708-81ce-05edc65e59c9.png)
```
Comment (Panquesito7) [inline README.md:11]:
```suggestion
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree)

![Screenshot](https://user-images.githubusercontent.com/60853067/133296120-dbdb1799-4cca-4708-81ce-05edc65e59c9.png)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/EddieHubCommunity/LinkFree/pull/397/conflicts). Let us know if you need any help here or in our [**Discord server**](http://discord.eddiehub.org). 🙂

**Comment by Dhananjaymahajan2001** [comment]
hlo when i opened this pull request readme page was different so now should i open new pr to make my changes or can it be done directly from this pr

**Comment by Panquesito7** [comment]
> hlo when i opened this pull request readme page was different so now should i open new pr to make my changes or can it be done directly from this pr

Try to make the changes in the same PR.

File: README.md (hunk 1 above)
Comment (krishguptadev) [reply README.md:10]: Open in gitpod seems fine enough

Comment (krishguptadev) [reply README.md:10]: For the screenshot part.. "A screenshot of the running application on a profile page"

**Comment by krishguptadev** [comment]
## Merge Conflict

Thank you for your contribution. Your pull request currently has conflicting changes with our `main` branch. In order to review this pull request, we will need those conflicts to be resolved.

We recommend doing this with the `git rebase` command. If you are unsure of how to fix this, let us know and we would be happy to assist you.

**Comment by Panquesito7** [comment]
> ## Merge Conflict
> Thank you for your contribution. Your pull request currently has conflicting changes with our `main` branch. In order to review this pull request, we will need those conflicts to be resolved.
> 
> We recommend doing this with the `git rebase` command. If you are unsure of how to fix this, let us know and we would be happy to assist you.

They can also take a look at the [**GitHub Docs**](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for that. 🙂

**Pull request closed without merging**